Source/HeadlessMain.cpp

```
@@ -0,0 +1,735 @@
+    bool ok{false};
+    std::string failure;
+    float maxPeak{0.0f};
+    float rms{0.0f};
+    uint64_t nanOrInfCount{0};
+    uint64_t hash64{0};
+    double renderMs{0.0};
+    double audioSeconds{0.0};
+    double cpuRatio{0.0};
+};
+
+static bool isBadSample(float v) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)